src/content/structured/components/select/code.mdx

```diff
@@ -2015,27 +2016,39 @@ return (
   },
 ];
 
+export const ReadonlyMulti = () => {
+  useEffect(() => {
+    document
+      .querySelector("#readonly-multi-select")
+      .setAttribute("readonly", "true");
+  }, []);
+  return (
+    <div style={{ display: "flex", flexDirection: "column", gap: "1rem" }}>
+      <IcSelect
+        label="What are your favourite types of coffee?"
+        multiple
+        options={[
+          { label: "Cappuccino", value: "Cap" },
+          { label: "Latte", value: "Lat" },
+          { label: "Americano", value: "Ame" },
+          { label: "Filter", value: "Fil" },
+          { label: "Flat white", value: "Fla" },
+          { label: "Mocha", value: "Moc" },
+          { label: "Macchiato", value: "Mac" },
+        ]}
+        value={["Cap", "Fla", "Moc"]}
+        readonly
```

Comment:
Might be good to remove this? Not a big deal but might be confusing if someone inspects the HTML and sees the two props